pkgs/development/interpreters/bwbasic/default.nix

```
@@ -0,0 +1,37 @@
+{ lib, stdenv, dos2unix, fetchzip }:
+
+stdenv.mkDerivation rec {
+  name = "bwbasic-3.20";
+
+  src = fetchzip {
+    url = "https://sourceforge.net/projects/bwbasic/files/bwbasic/version%203.20/bwbasic-3.20.zip/download";
+    extension = "zip";
+    stripRoot = false;
+    sha256 = "08z7cl6h9cikn714ld3804bnqcpk5z4sq8vqk5gvnd04n77ayyb9";
+  };
+
+  nativeBuildInputs = [ dos2unix ];
+
+  postPatch = ''
+    dos2unix configure
+    patchShebangs configure
+    chmod 700 configure
```

```suggestion
    chmod +x configure
```
I find this cleaner, also makes the intent more clear

---

Not my usual style, but yeah, it does seem clearer here. Done.